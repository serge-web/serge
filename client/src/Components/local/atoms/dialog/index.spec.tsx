/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import DialogModal from './index'

describe('Button component:', () => {
  it('renders correctly', () => {
    const onClose = (): void => {
      console.log('on modal close')
    }

    const onSave = (data: string): void => {
      console.log('on modal save: ', data)
    }
    const tree = renderer
      .create(
        <DialogModal open={false} title="Test" onClose={onClose} onSave={onSave} />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
