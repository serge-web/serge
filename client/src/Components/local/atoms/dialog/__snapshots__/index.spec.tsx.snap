// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component: renders correctly 1`] = `<div />`;
